docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -71,49 +59,71 @@ The [`download-data.py` script](https://github.com/AlexsLemonade/OpenScPCA-analy
-    Please download the full data release with caution!**
+    ```sh
+    ./download-data.py
+    ```
 
-```sh
-./download-data.py
-```
+- If you prefer to work with `AnnData` files, use the `--format` option as follows:
 
-If you prefer to work with `AnnData` files, use the `--format` option as follows:
+    ```sh
+    ./download-data.py --format AnnData
+    ```
 
-```sh
-./download-data.py --format AnnData
-```
+- To download a specific release other than the most recent release, you can use the `--release` option as follows.
+Again, please download full data releases with caution, as they are quite large!
+    - First, you may wish to use the `--list-releases` flag to see all available releases, which are named in `YYYY-MM-DD` format based on their release date.
 
-To review what samples would be downloaded without performing the download yet, you can use the `--dryrun` option as follows:
+    ```sh
+    # First, see which releases are available for download
+    ./download-data.py --list-releases
 
-```sh
-./download-data.py --dryrun
-```
+    # Download SingleCellExperiment format files for a specific release
+    # For example, this downloads the 2024-05-01 release
+    ./download-data.py --release 2024-05-01
 
-If you're only working with a subset of the data, you can use the `--projects` or `--samples` to download select samples (note: these options are mutually exclusive):
+    # Download AnnData format files for a specific release, for example
+    ./download-data.py --release 2024-05-01 --format AnnData
+    ```
 
-```sh
-# use the --projects option
-./download-data.py --projects SCPCPXXXXXX
+- If you're only working with a subset of the data, you can use the `--projects` or `--samples` option to download select projects or samples, respectively (note that these options are mutually exclusive):
 
-# use the --samples option
-./download-data.py --samples SCPCSXXXXXX,SCPCSXXXXXY
-```
+    ```sh
+    # use the --projects option
+    ./download-data.py --projects SCPCPXXXXXX
+
+    # use the --samples option
+    ./download-data.py --samples SCPCSXXXXXX,SCPCSXXXXXY
+    ```
+
+- To review the files would be downloaded without performing the download yet, you can use the `--dryrun` option as follows.
+    - We strongly recommend using the `--dryrun` flag the first time you run the script for a new data download to ensure the downloaded files are as expected.
+
+    ```sh
+    ./download-data.py --dryrun
+
+    # show what would be downloaded when the --projects option is used, for example
+    ./download-data.py --projects SCPCPXXXXXX --dryrun
+    ```
+
+The `download-data.py` script will always update the `data/current` symlink to point to the specified release directory.
+You can use the [`--update-symlink` flag described below](#updating-the-current-symlink) to update which release directory the `data/current` symlink directs to.
```

Comment:
This is not correct. It does _not_ update for specified releases, only when downloading the default `current` release or test data, see https://github.com/jashapiro/OpenScPCA-analysis/blob/e25539c402262ce7123791bc861703c191c1f067/download-data.py#L531-L533

```suggestion
The `download-data.py` script will update the `data/current` symlink to point to the current release directory when run without a `--release` option or to the test data directory when run with `--test-data`.
If you want to instead point to a specific release, you can use the [`--update-symlink` flag described below](#updating-the-current-symlink) to update which release directory the `data/current` symlink directs to.
```